source/clear-linux/get-started/virtual-machine-install/virtualbox-cl-installer.rst

```diff
@@ -3,256 +3,311 @@
+      :scale: 100%
+      :alt: File location and size
+
+      Figure 6: File location and size
 
 #. Click the *Create* button.
 
-#. A new virtual machine will be created and appear in the |VBM|. Click 
-   *Settings* to configure the |CL| VM.
+   A new virtual machine will be created and appear in the |VBM|.
 
-   .. image:: ./figures/vbox/vbox-vm-created.png
-      :alt: A VM selected in VirtualBox Manager
+#. Click *Settings*, or :guilabel:`gear` icon, to configure the |CL| VM.
 
-#. A *VM - Settings* window will appear. Navigate to the *System* pane from 
-   the left-hand and select the following setting:
+#. In the left-hand menu, navigate to the :guilabel:`System` menu and check
+   the following settings:
 
    - **Enable I/O APIC**
    - **Enable EFI (special OSes only)**
-   
 
-   .. image:: ./figures/vbox/vbox-vm-settings-EFI.png
-      :alt: Enable EFI on a VirtualBox VM settings
+   .. figure:: figures/vbox/virtualbox-cl-installer-07.png
+      :scale: 100%
+      :alt: Settings>System
+
+      Figure 7: Settings>System
 
 .. note::
-   By default, only 1 virtual CPU is allocated to the new VM. Consider 
-   increasing the number of virtual processors allocated to the virtual 
-   machine under Settings --> System --> Processor for increased 
+
+   By default, only 1 virtual CPU is allocated to the new VM. Consider
+   increasing the number of virtual processors allocated to the virtual
+   machine under Settings --> System --> Processor for increased
    performance.
 
+#. Select :guilabel:`OK`.
 
 Install |CL| on the |VB| VM
 ***************************
 
 |CL| is ready to be installed.
 
-
 Mount the installation ISO
 ==========================
-At this point, the newly created VM has a blank virtual hard disk with no 
-operating system.The |CL| installer ISO needs to be mounted as a virtual 
+
+At this point, the newly created VM has a blank virtual hard disk with no
+operating system. The |CL| installer ISO needs to be mounted as a virtual
 CD-ROM on the VM before powering the VM on.
 
-#. From the *VM - Settings* window, navigate to the *Storage* pane from the 
-   left-hand side.
+#. From the *ClearLinuxOs* :guilabel:`Settings` at left, select
+   :guilabel:`Storage`.
 
-#. From the middle *Storage Devices* column, click the blue CD disk labeled 
-   *Empty* under the *Controller: IDE* from.
+#. From :guilabel:`Storage Devices` in middle column, click the blue
+   disk labeled *Empty*.
 
-#. From the right-hand *Attributes* column, click the blue CD disk next to 
-   the *Optical Drive* drop down menu and click *Choose Virtual Optical 
+#. From the :guilabel:`Attributes` menu, click the blue CD disk next to
+   the *Optical Drive* drop down menu and click *Choose Virtual Optical
    Disk File...*
 
-   .. image:: ./figures/vbox/vbox-vm-settings-mount-ISO.png
-      :alt: Mounting an ISO in VirtualBox VM Settings  
+   .. figure:: figures/vbox/virtualbox-cl-installer-08.png
+      :scale: 120%
```

Comment:
Why are the images scaled up? Won't it look cleaner at 100%?

Reply:
Generally, yes. I attempted to make the wider images more legible, hence larger. I revised scale of 3 images back to 100% and revised Figure 12 in latest commit to make it more legible.